targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K82F/drivers/fsl_flash.c

```diff
@@ -402,7 +402,7 @@ static void (*callFlashCommonBitOperation)(FTFx_REG32_ACCESS_TYPE base,
  * @endcode
  * Note2: The binary code is generated by IAR 7.70.1
  */
-const static uint16_t s_flashRunCommandFunctionCode[] = {
```

Comment:
cc @mmahadevan108 please review this as it shall go upstream to fix warnings in drivers code